data_model/model/properties.mapping.js

```diff
@@ -179,8 +179,28 @@ export default ({ EXTENSIONS_KEY = '__extensions', FAKER_KEY = 'faker' } = {}) =
       ],
     },
   },
+  name_autocomplete: {
```

Comment:
this could apply to a lot more stuff than just `name`. if this field is generic, then anything can "copy_to" it. I would suggest calling this `quicksearch` and `copy_to: 'quicksearch'`

Reply:
i'll keep that in mind, but trying to get it working on name at least because it's not working at all. The phrase_prefix query 
```
{
  "query": {
    "bool": {
      "should": [
        {
          "bool": {
            "should": [
              {
                "multi_match": {
                  "query": "RGVF-tba",
                  "fields": [
                    "name_autocomplete"
                  ],
                  "type": "phrase_prefix"
                }
              }
            ]
          }
        }
      ]
    }
  }
}```
works but because `name_autocomplete` is a copy_to field it doesn't show up in _source, arranger graphql returns `null` for the field. Probably another instance of me using arranger wrong.